modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,85 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage"]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+Before you modify the machine set, you must manually generate the `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Configured your Azure portal account.
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+az login
+----
+. Create a Microsoft storage account and upload the `arm64` virtual hard disk (VHD) to your resource group:
++
+[source,terminal]
+----
+az storage account create -n <storage-account-name> -g <resource-group-name> -l westus --sku Standard_LRS 
+----
+. Create a storage container with your storage account using the following command: 
++
+[source,terminal]
++
+----
+az storage container create -n <container-name> --account-name <storage-account-name>
+----
+. Download the {op-system} Azure VHD from the `aarch64` link:https://mirror.openshift.com/pub/openshift-v4/arm64/dependencies/rhcos/pre-release/latest-4.11/[image mirror] page.
```

Comment:
@Prashanth684 Is there a way to use this image mirror page for the commands that reference RHCOS VHD and the blob name. Its all good If that doesn't work, I was thinking if there was a way to condense the procedure by using the mirror page (If not using the cli is easier here, Id rather do that)